server/auth/auth.go

```diff
@@ -326,8 +329,15 @@ func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) { /
 		log.Errorf("no client to penalize")
 		return
 	}
+	if auth.anarchy {
+		log.Infof("user %v penalized for rule %v, but not enforcing it", user, rule)
+		return
+	}
 	auth.storage.CloseAccount(client.acct.ID, rule)
 	client.conn.Banish() // May not want to do this. Leaving it for now.
```

Comment:
I would vote to remove the `Banish` here even without anarchy.

Reply:
I'm certain it should go.  Esp because we want users to continue active swaps.  I though I expanded on this in a comment somewhere but I'm not seeing it now.  Anyway, I'll comment it out with this PR.

Reply:
Oh, it was literally the next line. doh